.github/workflows/release.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'    # Match stable releases. e.g. v1.2.3
+      - 'v*.*.*-*'  # Match pre-releases. e.g. v1.2.3-beta, v1.2.3-rc.1
+
+permissions:
+  contents: read
```

Comment:
Is it read? [Here](https://github.com/hashicorp/terraform-provider-scaffolding-framework/blob/main/.github/workflows/release.yml) it mentions it should be write to create releases

Reply:
Line 17 takes precedence at the job level with write.